### PR TITLE
Use the compact signed proposal in consensus

### PR DESCRIPTION
**Pull request by yangby-cryptape**
Use the compact signed proposal in consensus.

### CODE REVIEW

File: communication.proto [hunk 1]
```diff
@@ -35,8 +35,6 @@ message InnerMessage {
         BlockTxHashes BlockTxHashes = 13;
         BlockTxHashesReq BlockTxHashesReq = 14;
 
-        VerifyTxReq VerifyTxReq = 15;
```
Comment (u2) [inline communication.proto]: Please add this back, refactor this later.

File: auth.proto [hunk 2]
```diff
@@ -32,13 +32,16 @@ message VerifyTxReq {
...
+    uint64 round = 2;
+    CompactBlock block = 3;
 }
 
 message VerifyBlockResp {
-    uint64 id = 1;
-    Ret ret = 2;
+    uint64 height = 1;
+    uint64 round = 2;
+    Ret ret = 3;
```
Comment (u2) [inline auth.proto]: `bool` is ok.

File: communication.proto (hunk 1 above)
Comment (yangby-cryptape) [reply communication.proto]: This line is useless.
`VerifyTxReq` can be removed from this `oneof`(`enum`) struct.

File: auth.proto (hunk 2 above)
Comment (yangby-cryptape) [reply auth.proto]: Yes. I delete `Ret` and add `bool pass` as the flag if the verification is passed or not.

**Pull request merged**